Server/controllers/User/logIn.js

```diff
@@ -0,0 +1,88 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+const { validateEmail } = require('../../helpers/Validation');
+
+const { SECRET } = process.env;
+const { logIn } = require('../../database/query/User/index');
```

Comment:
it is good, the just node you do not need to add index at end of URL

Reply:
Done